### DISCUSSION TITLE
Logs and QuarkusIntegrationTests

### DISCUSSION THREAD

**Discussion by gustoneves** [quarkusio/quarkus#22594] [Q&A] [answered]
During QuarkusIntegrationTests is it possible to view the logs of the application container during a mvn verify ? 

This is specially useful when running in a CI pipeline and want to see why a test failed.

[a comment by an automated account is not shown]

**Comment by geoand** [comment, marked as answer]
The application logs in a file named `quarkus.log` in the build system's outpur directory (`target` for Maven)

I also opened https://github.com/quarkusio/quarkus/pull/22596 to print the location of the file if there is a test error
